### ISSUE TITLE
Error trying to apply a light scene

### ISSUE THREAD

**Issue by thecode** [deconz-community/node-red-contrib-deconz#190]
### Current Behavior

Trying to call a light group scene from node-red raise the following error:
```txt
26 Dec 23:44:38 - [error] [deconz-output:Parents Room Fan Lights Night Mode] Error while processing command #1, TypeError: Cannot read properties of undefined (reading 'mode')
TypeError: Cannot read properties of undefined (reading 'mode')
    at CommandParser.getRequests (/config/node-red/node_modules/node-red-contrib-deconz/src/runtime/CommandParser.js:238:44)
    at /config/node-red/node_modules/node-red-contrib-deconz/nodes/out.js:182:4
```
The scene call fails and does not change the scene, added a flow json bellow
Pictures to show the setup:
![image](https://user-images.githubusercontent.com/1858925/147420754-83589068-d87a-46c7-ac62-ff20bb58f419.png)
![image](https://user-images.githubusercontent.com/1858925/147420764-3aadfcf5-c618-46ea-9502-51e33c30d596.png)
![image](https://user-images.githubusercontent.com/1858925/147420785-26dfbbf8-3b91-4edd-a29b-308dab7bd180.png)

### Expected Behavior

Call group scene should work :-)

### Steps To Reproduce

Create a group scene and call the scene

### Example flow

Sample flow with timestamp node and deconz output node
```json
[{"id":"a88f01d97540c6e3","type":"deconz-output","z":"9cfb01d0.1ec66","name":"Parents Room Fan Lights Night Mode","config_version":2,"server":"c37efbbc.780be8","statustext":"","statustext_type":"default","search_type":"device","device_list":["groups/device_id/1"],"device_name":"","query":"{}","commands":[{"arg":{"group":{"type":"num","value":"1"},"scene":{"type":"num","value":"1"},"retryonerror":{"type":"num","value":"0"},"aftererror":{"type":"continue","value":""}},"type":"deconz_state","domain":"scene_call","uniqueId":"ui-id-14"}],"specific":{"delay":{"type":"num","value":"50"},"result":{"type":"at_end","value":""}},"x":730,"y":1740,"wires":[[]]},{"id":"772908b603987adc","type":"inject","z":"9cfb01d0.1ec66","name":"","repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"","payloadType":"date","x":480,"y":1740,"wires":[["a88f01d97540c6e3"]]},{"id":"c37efbbc.780be8","type":"deconz-server","name":"Phoscon-GW","ip":"core-deconz","port":"40850","ws_port":"8081","secure":false,"polling":""}]
```

### Environment

- deCONZ plugin version: 2.2.0
- Node-RED version: v2.1.4
- Node.js version:  v16.13.1
- npm version: 8.1.3
- Platform/OS: Linux 5.10.83 x64 LE (using Home assistant addon, version 10.2.2)
- Browser: Chrome, Version 96.0.4664.110 (Official Build) (64-bit)

**Comment by Zehir** [comment]
The UI on your screenshoot should have a new field between the type and the picker.
Did you refresh your browser ?

**Comment by thecode** [comment]
Thanks for the fast reply, 
I did refresh the browser before, **using Chrome "Empty Cache and hard reload" solve the issue.** 
Adding the picture with the new field if others encounter this issue also:
![image](https://user-images.githubusercontent.com/1858925/147421083-e0a9233b-0f16-4f9e-9757-df079c2c28ec.png)

**Comment by Zehir** [comment]
The auto migration feature should have handle this case. Did you save this node with the wrong UI ?

**Comment by Zehir** [comment]
I also found a side issue when importing your flow but it's a really edge case when you try to load the scene picker when your configuration is invalid.

**Comment by thecode** [comment]
I added this node after updating the plugin to 2.2.0, restarted node-red and reloaded the page (just closed and opened again did not try to force refresh it)
The comment in the 2.2.0 change log reminded me that I wanted to use Deconz scenes for a long time so I decided to check it

**Comment by Zehir** [comment]
> I added this node after updating the plugin to 2.2.0, restarted node-red and reloaded the page (just closed and opened again did not try to force refresh it) The comment in the 2.2.0 change log reminded me that I wanted to use Deconz scenes for a long time so I decided to check it

ok that why this throw errors ;)

**Comment by Zehir** [comment]
Ok now the user won't be able to reproduce this beavior. It's fixed in 2.2.1
